### DISCUSSION TITLE
I disabled the HTML Language Features.

### DISCUSSION THREAD

**Discussion by ivanjeremic** [codesandbox/codesandbox-client#6755] [General]
Hello I have disabled the HTML Language Features inside Extensions > Running Extension (right click & disable).

Now the official HTML Language Features are not in the list anymore how to enable them back?

**Comment by JamesACS** [comment]
Hey @ivanjeremic - Please could you provide more specifics? Is this for the Chrome extension we offer?

**Reply by ivanjeremic** [reply]
No not an extension I talk about codesandbox.io

**Reply by JamesACS** [reply]
Sorry, I'm a little confused. Could you walk us through what happened in some more detail?
